e2e/ctl_v3_test.go

```diff
@@ -155,27 +168,61 @@ func testCtl(t *testing.T, testFunc func(ctlCtx), opts ...ctlOption) {
 	}
 }
 
+var envMap = make(map[string]struct{})
+
+func init() {
+	// Keep up-to-date with 'prefixArgs'
+	fs := []string{
+		"endpoints",
+		"dial-timeout",
+		"insecure-transport",
+		"insecure-skip-tls-verify",
+		"cacert",
+		"cert",
+		"key",
+		"user",
+	}
+	for _, f := range fs {
+		envMap[flags.FlagToEnv("ETCDCTL", f)] = struct{}{}
+	}
+}
```

Comment:
i really do not like the global map. update the fmap should ok since fmap itself is a pointer to a struct. can you give it a try again?